File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0) return false; //all negative numbers are not palindrome
        
        string str = to_string(x);
        string reversedStr = str;
        
        reverse(reversedStr.begin(), reversedStr.end());
        
        if(str != reversedStr) return false;
        else return true;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if (x < 0):
            return False
        
        string = str(x)
        reverse = string[::-1]
        
        if(string == reverse):
            return True
        else:
            return False